### ISSUE TITLE
Remove opacity on body

### ISSUE THREAD

**Issue by jzhu01** [Webscaper/prj-rev-bwfs-tea-cozy#3]
Seen here: https://github.com/Webscaper/prj-rev-bwfs-tea-cozy/blob/master/Tea%20Cozy/resources/css/style.css#L11

Since the opacity is making the text darker and harder to read, I recommend removing it. I would suggest adding it to the image, to make it easier to read the overlaying text.

**Comment by Webscaper** [comment]
Perhaps I misread the instructions of the project. Top left on the redline image It mentions opacity 0.9.

**Comment by Webscaper** [comment]
With that said, I wholeheartedly agree

**Comment by jzhu01** [comment]
Yup, that part of the instruction is a bit vague, I do admit. I'm glad you agree! :)
